uiServices/test/godog_test.go

```diff
@@ -292,7 +294,7 @@ func (a *apiFeature) iSendRequestToWithBody(method, endpoint string, body *gherk
 	return nil
 }
 
-func makePutRequest(client *http.Client, method, endpoint, body string, header string) (*http.Response, error) {
+func makePutRequest(client *http.Client, method, endpoint, body string, headerPath string, headerName string) (*http.Response, error) {
```

Comment:
you can replace the params with 
`client *http.Client, method, endpoint, body, headerPath, headerName string`